v3/src/components/graph/adornments/v2-adornment-importer.ts

```diff
@@ -322,12 +321,13 @@ export const v2AdornmentImporter = ({data, plotModels, attributeDescriptions, yA
   }
 
   // MOVABLE VALUES
-  const movableValuesAdornment = v2Adornments.multipleMovableValues
+  const movableValuesAdornment = v2Adornments?.multipleMovableValues
   if (movableValuesAdornment) {
     const values: Record<string, any> = {}
     instanceKeys?.forEach((key: string) => {
       const plotValues: number[] = []
-      movableValuesAdornment.valueModels.forEach((valueModel: Record<string, any>) => {
+      const valueModels = movableValuesAdornment.valueModels ?? movableValuesAdornment.values
```

Comment:
I was running into this same problem yesterday and tracked down several documents with `movableValuesAdornment.values`. From all of the documents I looked at, the type of `movableValuesAdornment.values` was `{isVisible: boolean, value: number}[]`. So there would be no `valueModel.values` and `valueModel.values._main` would then crash. 

If you want to just ignore the `movableValuesAdornment.values` in this PR, I can fix it in mine.

Reply:
I added a commit which updates the types, removes the handling of legacy `values`, and adds a comment about the need to handle legacy `values`.